packages/reference/src/components/LinkActions/LinkEntityActions.tsx

```diff
@@ -88,37 +88,31 @@ export function useLinkActionsProps(props: LinkEntityActionsProps): LinkActionsP
     [sdk, entityType, onCreated]
   );
 
-  const onLinkExisting = React.useCallback(
-    async (index?: number) => {
-      const entity = await selectSingleEntity({
-        sdk,
-        entityType,
-        editorPermissions,
-      });
-      if (!entity) {
-        return;
-      }
+  const onLinkExisting = async (index?: number) => {
+    const entity = await selectSingleEntity({
+      sdk,
+      entityType,
+      editorPermissions,
+    });
+    if (!entity) {
+      return;
+    }
 
-      onLinkedExisting([entity], index);
-    },
-    [sdk, entityType, onLinkedExisting]
-  );
+    onLinkedExisting([entity], index);
+  };
 
-  const onLinkSeveralExisting = React.useCallback(
-    async (index?: number) => {
-      const entities = await selectMultipleEntities({
-        sdk,
-        entityType,
-        editorPermissions,
-      });
+  const onLinkSeveralExisting = async (index?: number) => {
+    const entities = await selectMultipleEntities({
+      sdk,
+      entityType,
+      editorPermissions,
+    });
 
-      if (!entities || entities.length === 0) {
-        return;
-      }
-      onLinkedExisting(entities, index);
-    },
-    [sdk, entityType, onLinkedExisting]
-  );
+    if (!entities || entities.length === 0) {
+      return;
+    }
+    onLinkedExisting(entities, index);
+  };
 
   return useMemo(
```

Comment:
Your changes look suspicious to me. Doing that means that this `useMemo` doesn't make sense anymore. If this is a hotfix, I would add a note. Something like `// Use raw callbacks to avoid any memoization`.

if it's urgent, how about a follow-up ticket to re-add `useCallback` and find out why this logic broke in the first place (which rerenders and recalculations are desired and which are not).

@IBratWurst this might be related to the hotfix you introduced recently for the other bug.

Reply:
when used correctly, memoized callbacks shouldn't case any issue or need to be removed. The reason for that should be that memoization causes 0 or negative performance gain.

Anyway I don't think this interferes with the hotfix for the disabled link button

Reply:
Thanks for checking 👍 (I thought it related to an empty list of content types)